### DISCUSSION TITLE
Home Assistant Install....

### DISCUSSION THREAD

**Discussion by doug62** [mKeRix/room-assistant#1101] [Q&A]
I'm assuming their is an HA integration - I checked HACs, Add-on store and docs for integration repo URLs but can't find one.

**Comment by phoniclynx** [comment]
It just appears there.. search for "Trackers"

**Comment by odhiambo** [comment]
Follow this howto for the addon:

https://peyanski.com/presence-detection-using-room-assistant-home-assistant-and-raspberry-pi/

**Comment by mKeRix** [comment]
room-assistant does not have a separate integration you need to install - you just need to activate the MQTT integration with auto-discovery support in HA. If you want to run it on Home Assistant OS you additionally have the option to install the [add-on](https://www.room-assistant.io/guide/installation.html#running-with-home-assistant-os).
